pycds/weather_anomaly.py

```diff
@@ -23,20 +23,50 @@
   - These views are defined with plain-text SQL queries instead of with SQLAlchemy select expressions.
       The SQL SELECT statements were already written, and the work required to translate them to SQLAlchemy seemed
       excessive and unnecessary. See https://docs.sqlalchemy.org/en/latest/core/tutorial.html#using-textual-sql
+
+Materialized View: Monthly total precipitation (MonthlyTotalPrecipitation)
+  - Observations flagged with meta_native_flag.discard or meta_pcic_flag.discard are not included in the view.
+  - data_coverage is the fraction of observations actually available in a month relative to those potentially
+     available in a month. This computation is correct if and only if the observation frequency does not change
+     during any one day in the month. It remains approximately correct if such days are rare, and remains valid
+     for the purpose of distinguishing adequate coverage.
+  - This view is defined with plain-text SQL queries instead of with SQLAlchemy select expressions.
+      The SQL SELECT statements were already written, and the work required to translate them to SQLAlchemy seemed
+      excessive and unnecessary. See https://docs.sqlalchemy.org/en/latest/core/tutorial.html#using-textual-sql
+
 """
 
 import sqlalchemy
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.sql import text, column
 from sqlalchemy.schema import DDL
 
-from pycds.view_helpers import ViewMixin  # TODO: Remove
+from pycds.view_helpers import ViewMixin
 from pycds.materialized_view_helpers import MaterializedViewMixin
 
 Base = declarative_base()
 metadata = Base.metadata
 
 
+class DiscardedObsId(Base, ViewMixin):
```

Comment:
Yeah, this is a decent way to go about this. One small suggestion: I'd name the class ``DiscardedObs`` with an attribute of ``id``. Otherwise, you'll be saying ``DiscardedObsId.id`` all over the place.